PRChecker/Shared/Unique Views/ContentView.swift

```diff
@@ -37,7 +38,26 @@ struct ContentView: View {
                 })
             }
             ToolbarItem(placement: .primaryAction) {
-                Button{
+                if refreshable {
```

Comment:
Would names like `isRefreshing`/`isLoading` more like status names?

Reply:
Ah, good point. I'll change that.